image_based_install_and_upgrade/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-generate-seed.adoc

```diff
@@ -0,0 +1,20 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cnf-image-based-upgrade-generate-seed-image"]
+= Generating a seed image for the image-based upgrade with the {lcao}
+include::_attributes/common-attributes.adoc[]
+:context: generate-seed
+
+toc::[]
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` custom resource (CR).
```

Comment:
Is this required? If you are looking for a lead-in sentence maybe the sentence could have more details or be a little more descriptive. 
Also, convert the sentence into an assertive one since this is an assembly.